ament_tools/helper.py

```diff
@@ -222,17 +222,37 @@ def extract_argument_group(args, delimiting_option):
     return trimmed_args, extracted_args
 
 
-def deploy_file(context, source_base_path, filename, dst_subfolder='', executable=False):
+def deploy_file(
+    context,
+    source_base_path,
+    filename,
+    dst_subfolder='',
+    executable=False,
+    skip_if_exists=False
+):
+    # copy the file if not already there and identical
+    source_path = os.path.join(source_base_path, filename)
+
     # create destination folder if necessary
     destination_path = os.path.join(
         context.install_space, dst_subfolder, filename)
+    # If the file exists and we should skip if we didn't install it.
+    if os.path.exists(destination_path) and skip_if_exists:
```

Comment:
Ok, I updated to check to see if it exists or if it is a link. Should that be sufficient?